src/validation_workflow/api_test_cases.py

```diff
@@ -28,10 +28,10 @@ def test_cases(self) -> Any:
 
         # the test case parameters are formated as ['<request_url>',<success_status_code>,'<validate_string(optional)>']
         test_cases = [
-            ['https://localhost:9200/', 200, '"number" : "' + self.opensearch_image_version + '"'],
-            ['https://localhost:9200/_cat/plugins?v', 200, ''],
-            ['https://localhost:9200/_cat/health?v', 200, 'green'],
-            ['http://localhost:5601/api/status', 200, '"number":"' + self.opensearch_dashboards_image_version + '"']
+            ['https://localhost:9200/', 0, '"number" : "' + self.opensearch_image_version + '"'],
+            ['https://localhost:9200/_cat/plugins?v', 0, ''],
+            ['https://localhost:9200/_cat/health?v', 0, 'green'],
+            ['http://localhost:5601/api/status', 0, '"number":"' + self.opensearch_dashboards_image_version + '"']
```

Comment:
This doesnt make sense even though I understand you are just using the `return 0` value before.
If this is the case you should just take 200, and make sure exit is without error if return is 200.
Not trying to change the status code 200 to 0.

Reply:
Oh... the reason to replace '200' with '0' is because when using the library 'requests', the API does return the HTTP code ; however, in using the Curl, the return code is '0' representing the successful response.

Reply:
my bad ,,, I should have reserached it more ... @gaiksaya just send me a link showing how to get HTTP status code in Curl ... I will make relevant changes and revert it back to 200 here ,, thanks!

Reply:
@peterzhuamazon .. I have it updated with status code 200 by using '-i' to get the HTTP response header in the Curl .. it looks working. thanks!!